deploy-v2.js

```diff
@@ -0,0 +1,162 @@
+};
+
+const DeployBountyToken = async (tokenName, tokenSymbol) => {
+  const deployer = new BountyToken(connection);
+
+  await deployer.loadAbi();
+
+  const tx = await deployer.deployJsonAbi(
+    tokenName,
+    tokenSymbol
+  );
+
+  return tx.contractAddress;
+};
+
+const DeployNetwork_v2 = async (settlerAddress, nftAddress, nftUri) => {
+  const deployer = new Network_v2(connection);
+
+  await deployer.loadAbi();
+
+  const tx = await deployer.deployJsonAbi(settlerAddress, nftAddress, nftUri);
+
+  return tx.contractAddress;
+};
+
+const DeployNetworkFactoryV2 = async (settlerToken) => {
+  const deployer = new NetworkFactoryV2(connection);
+
+  await deployer.loadAbi();
+
+  const tx = await deployer.deployJsonAbi(settlerToken);
+
+  return tx.contractAddress;
+}
+
+const DeployNetworkRegistry = async (token) => {
+  const deployer = new Network_Registry(connection);
+
+  await deployer.loadAbi();
+
+  const tx = await deployer.deployJsonAbi(token, 1000000, await connection.getAddress(), 10000);
+
+  return tx.contractAddress;
+}
+
+const InitialDeploy = async () => {
+  await connection.start();
+
+  console.log("Deploying contracts...");
+  const settlerAddress = await DeployERC20("Bepro Network", "BEPRO", cap);
+  const transactionalAddress = await DeployERC20("USD Coin", "USDC", cap);
+  const rewardAddress = await DeployERC20("MVLS Network", "MVLS", cap);
+  const bountyTokenAddress = await DeployBountyToken("NFT Token", "NFT");
+  const networkAddress = await DeployNetwork_v2(settlerAddress, bountyTokenAddress, "//");
+  const registryAddress = await DeployNetworkRegistry(settlerAddress);
+  console.log("Done.");
+
+  const settler = new ERC20(connection, settlerAddress);
+  const bountyToken = new BountyToken(connection, bountyTokenAddress);
+  const transactional = new ERC20(connection, transactionalAddress);
+  const reward = new ERC20(connection, rewardAddress);
+  const registry = new Network_Registry(connection, registryAddress);
+
+  await settler.loadContract();
+  await bountyToken.loadContract();
+  await transactional.loadContract();
+  await reward.loadContract();
+  await registry.loadContract();
+
+  console.log("Registering Network...");
+  // Approve, lock and create a new network using the factory
+  await settler.approve(registryAddress, 1000000);
+  await registry.lock(1000000);
+  await registry.registerNetwork(networkAddress);
+  console.log("Done.");
+
+  const networkObj = new Network_v2(connection, networkAddress);
+
+  await networkObj.loadContract();
+
+  console.log("Setting dispatcher...");
+  await bountyToken.setDispatcher(networkAddress);
+  console.log("Done.");
+
+  console.log("Changing parameters...");
+  await networkObj.changeDraftTime(61);
+  await networkObj.changeDisputableTime(120);
+  await networkObj.changeCouncilAmount(101000);
+  console.log("Done.");
+
+  const address1 = "0xf3ad222f9Aa841C240022b73cA228A71a92fF82D";
+  const address2 = "0xC3869b5a2D0EA422a007a2503C0d616C6a8d6B63";
```

Comment:
make this addresses and array, and then iterate over them,
```ts
await Promise.all(
  ['address', 'address']
    .map(address => {
      await settler.transferTokenAmount(address, 10000000)
      await transactional.transferTokenAmount(address, 10000000);
      await reward.transferTokenAmount(address, 10000000);
    }):
```

PS: Make sure to only do this if we haven't provided these values, because if we have provided these contract addresses (reward, transactional, settler) then these are already existing coins and most likely _the person running the script_ won't want to transfer more things.